Source/Application.cpp

```diff
@@ -81,12 +81,8 @@ bool Application::Start()
 
 update_status Application::Update()
 {
-	float ms;
-#ifdef ENGINE
-	(isOnPlayMode) ? ms = onPlayTimer.Read() : ms = appTimer.Read();
-#else
-	ms = appTimer.Read();
-#endif // ENGINE
+	bool playMode = isOnPlayMode;
```

Comment:
Even if it's redundant, maybe we could change this to
```suggestion
#ifdef ENGINE
	bool playMode = isOnPlayMode;
#else
	bool playMode = false;
#endif
```
Just so it's obvious to the reader what will happen in Game. But as I said it's redundant, since `isOnPlayMode` will always be `false` in Game.

Reply:
Actually it should be "playMode = true" when it's in game mode. But even then, i woudln't add this change, not a fan of having redundant code xD and also the boolean is self explenatory i think it's pretty clear as it is.